.github/workflows/build_wheels_linux.yml

```diff
@@ -0,0 +1,39 @@
+name: Trigger Linux Wheels Build for Nightlies/Release
+
+on:
+  pull_request:
+    paths: .github/workflows/build_wheels_linux.yml
+  push:
+    branches:
+      - nightly
```

Comment:
As I understand, it should be triggered by the scheduler once a day, right? Or do we have some kind of automatic push to the night branch?

Reply:
We have an automatic push from an internal cron job